### DISCUSSION TITLE
월드로 제페토 맵 만들며 궁금증있어서 올립니다

### DISCUSSION THREAD

**Discussion by ChanjongB** [naverz/zepeto-studio-kor#455] [5. Q&A]
월드로 맵을 만들려하는데 궁금증이 많아서 질문드려요
터레인 텍스쳐 같은경우 제페토에서 쓰는 기본만사용해야하나요?
툴같은경우도 terrain tools Sample asset pack  가있는데  사용할수있나요? 
유료 또는 무료  심플 폴리곤 어셋같은경우 사용할수있나요?
가이드 사항에는 있는데 못찾는건지 없는건지 몰라올립니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 유니티 개발시 사용할 수 있는 모든 어셋은 사용 가능합니다. 
다만 에셋 팩에 C# 코드가 들어있다면 동작하지 않는다는 점 참고 부탁드립니다.
감사합니다.
